### ISSUE TITLE
Make sure this works on arm v7

### ISSUE THREAD

**Issue by Licenser** [simd-lite/simd-lite#5]
Right now we only check for aarch64 (given that's the architecture we can test on) it'd be good to validate everything works on a good old arm device or even run that in CI.

**Comment by sunnygleason** [comment]
Good call! Can we set up another Arm CI build that uses the v7 feature set (or targets another CPU) instead?
